package_index.template

```diff
@@ -0,0 +1,55 @@
+        {
+          "name": "imgtool",
+          "version": "%%VERSION%%",
+          "systems": [
+            {
+              "host": "i386-apple-darwin11",
+              "url": "http://downloads.arduino.cc/tools/%%FILENAME_MACOS_64BIT%%",
```

Comment:
Do you know whether there are specific reasons for using `http` vs `https` in the download URLs of the package index? I see most of them are `http`, but not all.

I am interested in the subject because two users reported Boards Manager installation failures on the forum today and in the end they found it only succeeded once they changed the openOCD tool download URL to use `https` (the standard URL worked fine for me).